code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -25,8 +25,10 @@
 	return
 
 /obj/item/rig_module/proc/say_to_wearer(var/string)
-	ASSERT(rig.wearer)
-	to_chat(rig.wearer, "\The [src] reports: <span class = 'binaryradio'>[string]</span>")
+	if(!ishuman(rig.wearer))
+		return
+	var/mob/living/carbon/human/H = rig.wearer
```

Comment:
there's no reason to cast this to human as far as I can tell

Reply:
i guess